server/src/server.ts

```diff
@@ -105,77 +111,76 @@ connection.onDefinition((async(textPosition) => {
+  return handleErrors(autoComplete(textPosition));
+});
+
+async function autoComplete(textPosition: TextDocumentPositionParams):
+    Promise<CompletionList|undefined> {
+      const localPath = getWorkspacePathToFile(textPosition.textDocument);
+      if (!localPath || !editorService) {
+        return;
+      }
+      const completions: (TypeaheadCompletion|undefined) =
+          await editorService.getTypeaheadCompletionsAtPosition(
+              localPath, convertPosition(textPosition.position));
+      if (!completions) {
+        return;
+      }
+      if (completions.kind === 'element-tags') {
+        return {
+          isIncomplete: false,
+          items: completions.elements.map(c => {
+            return <CompletionItem>{
+              label: `<${c.tagname}>`,
+              kind: CompletionItemKind.Class,
+              documentation: c.description,
+              insertText: c.expandTo
+            };
+          }),
         };
-      }),
-    };
-  } else if (completions.kind === 'attributes') {
-    return {
-      isIncomplete: false,
-      items: completions.attributes.map(a => {
-        const item: CompletionItem = {
-          label: a.name,
-          kind: CompletionItemKind.Field,
-          documentation: a.description,
-          sortText: a.sortKey
+      } else if (completions.kind === 'attributes') {
+        return {
+          isIncomplete: false,
+          items: completions.attributes.map(a => {
+            const item: CompletionItem = {
+              label: a.name,
+              kind: CompletionItemKind.Field,
+              documentation: a.description,
+              sortText: a.sortKey
+            };
+            if (a.type) {
+              item.detail = `{${a.type}}`;
+            }
+            if (a.inheritedFrom) {
+              if (item.detail) {
+                item.detail = `${item.detail} ⊃ ${a.inheritedFrom}`;
+              } else {
+                item.detail = `⊃ ${a.inheritedFrom}`;
+              }
+            }
+            return item;
+          }),
         };
-        if (a.type) {
-          item.detail = `{${a.type}}`;
-        }
-        if (a.inheritedFrom) {
-          if (item.detail) {
-            item.detail = `${item.detail} ⊃ ${a.inheritedFrom}`;
-          } else {
-            item.detail = `⊃ ${a.inheritedFrom}`;
-          }
-        }
-        return item;
-      }),
-    };
-  } /* else if (completions.kind === 'resource-paths') {
-     return {
-       isIncomplete: false,
-       items: completions.paths.map(p => {
-         const item: CompletionItem = {label: p, kind: CompletionItemKind.File};
-         if (completions.prefix && p.startsWith(completions.prefix)) {
-           // Only insert text to complete the text typed so far.
-           item.insertText = p.substring(completions.prefix.length);
-         }
-         return item;
-       })
-     };
-   } */
-});
+      } /* else if (completions.kind === 'resource-paths') {

```

Comment:
It was commented out with the thought that it would be re-enabled quickly. We ended up going back to the drawing board there, so it'll probably be a little while there. Probably best to just delete it.

Good night, sweet path completions. You'll always be in our hearts, and in our source control.